botstrap.py

```diff
@@ -126,6 +179,24 @@ def create_webhook(name: str, channel_id_: int, client: DiscordClient) -> str:
 
     config_str += "\n#Channels\n"
 
+    rules_channel_id = all_channels[RULES_CHANNEL_NAME]
+    announcements_channel_id = all_channels[ANNOUNCEMENTS_CHANNEL_NAME]
+
+    upgrade_server_to_community_if_necessary(GUILD_ID, rules_channel_id, announcements_channel_id, discord_client)
+
+    create_help_channel = True
+
+    if PYTHON_HELP_CHANNEL_NAME in all_channels:
+        python_help_channel_id = all_channels[PYTHON_HELP_CHANNEL_NAME]
+        if not is_forum_channel(python_help_channel_id, discord_client):
+            delete_channel(python_help_channel_id, discord_client)
+        else:
+            create_help_channel = False
+
+    if create_help_channel:
+        python_help_channel_name = PYTHON_HELP_CHANNEL_NAME.replace('_', '-')
+        python_help_channel_id = create_forum_channel(python_help_channel_name, GUILD_ID, discord_client)
```

Comment:
Also, why are you not passing the category ID here? you already wrote the function to take one.

Reply:
The category wasn't created in the test server at first, and wasn't tracked.

Idk tbh, but here it is
e4903f80dc409d6ebee7170be8ee5e38c242d81d